app/models/stripe_authorization.py

```diff
@@ -14,7 +14,7 @@ class StripeAuthorization(db.Model):
     stripe_refresh_token = db.Column(db.String)
     stripe_publishable_key = db.Column(db.String)
     stripe_user_id = db.Column(db.String)
-    stripe_email = db.Column(db.String)
+    stripe_auth_code = db.column(db.String)
```

Comment:
Is this a typo? Shouldn't it be `db.Column`?

Reply:
Yes,
Would have found out while migrating the db but thanks :)